app/views/questions/_questions.html.erb

```diff
@@ -22,7 +22,12 @@
           <span id="tag_<%= tag.id %>" class="label label-primary"><a href="/questions/tag/<%= tag.name %>"><%= tag.name %></a></span>
         <% end %>
         </span>
-        <p style="margin-top:30px;"><% if logged_in_as(['admin','moderator']) %><a class="btn btn-default btn-xs" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> Spam</a><% end %>
+        <p style="margin-top:30px;">
```

Comment:
Due to recent changes by Kevin and design suggested by Jeff, we need to change this code a lot. So, I am closing this PR. @okonek please feel free to open up a PR for the design suggested by Jeff () and then a follow up PR for the linking of modals(#4255).

Reply:
These changes are no longer needed due to new PRs merged.

Reply:
Please feel free to open a new pr @okonek 
Thanks for awesome work you are doing at PL